openedx/core/djangoapps/discussions/models.py

```diff
@@ -126,12 +158,15 @@ class Features(Enum):
             Features.USER_MENTIONS.value,
         ],
         'external_links': ProviderExternalLinks(
-            'https://www.youtube.com/watch?v=ZACief-qMwY',
-            '',
-            'https://hubs.ly/H0J5Bn70',
-            '',
-            'learnmore@yellowdig.com',
+            learn_more='https://www.youtube.com/watch?v=ZACief-qMwY',
+            configuration='',
+            general='https://hubs.ly/H0J5Bn70',
+            accessibility='',
+            contact_email='learnmore@yellowdig.com',
         )._asdict(),
+        "messages": ProviderMessages(
```

Comment:
If you opt to use the classless syntax, this can up upgraded to 

`ProviderMessages(messages=[pii_sharing_required_message("Yellowdig")])`

Reply:
also, we don't the datastruct should simply be `provider.messages[]` not something `provider.messages.setup[]`. This should be simplified as well.

Reply:
Done; since we simplified this to just an array, I've removed the `ProviderMessages` tuple.